src/bootstrap/bootstrap.py

```diff
@@ -236,6 +237,11 @@ def default_build_triple():
         if ostype.endswith('WOW64'):
             cputype = 'x86_64'
         ostype = 'pc-windows-gnu'
+    elif sys.platform == 'win32':
+        # Some Windows platforms might have a `uname` command that returns a
+        # non-standard string (e.g. gnuwin32 tools returns `windows32`). In
+        # these cases, fall back to using sys.platform.
+        return 'x86_64-pc-windows-msvc'
```

Comment:
Hm, is there a reason not to directly compare with windows32 here? (FWIW windows32 to me sounds like a 32-bit platform, but maybe that's historical baggage or something?)

Reply:
I think that's historical baggage of the gnuwin32 tools (they also return `i686-pc` for `uname -m` even on a 64-bit system). I didn't find a 64-bit equivalent, so this is probably not a reliable way to determine the actual architecture.

My thinking here was to provide a default for Windows platforms with any non-standard `uname` value (same as not having `uname`), but I can certainly change this to a direct comparison with windows32. What would be best?

Reply:
Hm, so I think this is probably fine -- we can always limit the scope later, if necessary, or add more conditionals.

Reply:
Thanks for the review!